apothecary/formulas/rtAudio/rtAudio.sh

```diff
@@ -60,6 +60,7 @@ function build() {
 					 -Iinclude \
 					 -DHAVE_GETTIMEOFDAY \
 					 -D__MACOSX_CORE__ \
+					 -mmacosx-version-min=10.9 \
```

Comment:
Thanks!
Could you change `10.9` to `${OSX_MIN_SDK_VER}` ( which is currently set to 10.9 but allows us to easily update in the future )